controller/controller_pokemons.go

```diff
@@ -7,7 +7,7 @@ import (
 	"strconv"
 
 	"github.com/gorilla/mux"
```

Comment:
For best practice separate the import libraries like:

Import(
// Standar library packages

// Our packages

// Third party libraries

)